builds/fabric8-che/plugins/che-plugin-bayesian-lang-server/src/main/java/org/eclipse/che/plugin/languageserver/bayesian/server/launcher/BayesianLanguageServerLauncher.java

```diff
@@ -87,8 +87,10 @@ private static LanguageServerDescription createServerDescription() {
                                                                               null),//
                                                            new DocumentFilter(JsonModule.LANGUAGE_ID,//
                                                                               "package\\.json",//
-                                                                              null)//
-                                             // TODO pom.xml
+                                                                              null),
+                                                           new DocumentFilter("pom",
```

Comment:
Well, in this case we'll have to wait for the upstream PR to be merged first.
I'd prefer doing this in a second PR synchronized with a related upstream Che PR.
Is it OK for you ?